src/components/App/index.jsx

```diff
@@ -101,8 +102,15 @@ const AppWrapper = () => {
       return false;
     }
     const startTimestamp = config.MAINTENANCE_ALERT_START_TIMESTAMP;
+    const endTimeStamp = config.MAINTENANCE_ALERT_STOP_TIMESTAMP;
```

Comment:
nit: similar casing for `Timestamp` as in `startTimestamp`.

nit x2: any concerns around using "end" here but "stop" within the config setting? I might recommend changing the config setting to `MAINTENANCE_ALERT_END_TIMESTAMP` and keeping `endTimestamp` 🤔 (using "end" is more parity with other end-related data fields in other configs/APIs/etc.)